Product/Production/Adapters/General/CONNECTAdminGUI/pom.xml

```diff
@@ -231,7 +231,7 @@
     <dependency>
       <groupId>commons-fileupload</groupId>
       <artifactId>commons-fileupload</artifactId>
-      <scope>runtime</scope>
+       <scope>runtime</scope>
```

Comment:
If I add version in original code in line 664... it gives error... you can try yourself